tests/Guzzle7/phpstan.neon

```diff
@@ -6,5 +6,9 @@ parameters:
 	bootstrapFiles:
 		- vendor/autoload.php
 	reportUnmatchedIgnoredErrors: false
+	ignoreErrors:
+		-
+			message: '#^Invalid type Http\\Client\\Exception to throw.$#'
```

Comment:
FYI @Quetzacoalt91, we choosed (with @Matt75) to ignore those errors rather than import all the Exceptions related, like :
- TransferException
- NetworkException
- HttpException
- RequestException

Reply:
Yeah, I'm not talking about importing all the classes but the interface.

https://github.com/php-fig/http-client/blob/master/src/ClientExceptionInterface.php

Reply:
Yes, it was only to give you the explanation about the fact we ignore those errors in phpstan-g7.
I added the interface that you're talking about.